### DISCUSSION TITLE
Avatar Component

### DISCUSSION THREAD

**Discussion by qscheitlin** [UI5/webcomponents-react#4582] [Q&A] [answered]
Hello, 

I am experiencing a little bit of UI flickering when I am asynchronously downloading an avatar image for the shell bar.  I saw in the documentation the below statements.  I tried creating my own css class based on the below, but it didn't seem to have any affect.  My question is if anyone knows of an example sitting around that I could reference as a guide

"Note: If you experience flickering of the provided image, you can hide the component until it is being defined with the following CSS:

Avatar:not(:defined) {  visibility: hidden; }"

**Comment by MarcusNotheis** [comment, marked as answer]
Hey @qscheitlin,

looks like our script is replacing too much tag names 😄 
This actually should look like:

```css
[ui5-avatar]:not(:defined) {
  visibility: hidden;
}
```
